webpack/dev.config.js

```diff
@@ -0,0 +1,7 @@
+const WebpackNotifierPlugin = require('webpack-notifier');
```

Comment:
'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).<br>'require' is not defined.